index.html

```diff
@@ -6,7 +6,13 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="content-type" content="text/html; charset=UTF8">
-
+    <!--Adding meta Tag for search engine optimisation-->
+    <meta name="Description" content="We especially welcome contributions from people from groups underrepresented in free and open source software!">
+    <meta property="og:site_name" content="code.publiclab.org">
+    <meta property="og:url" content="https://code.publiclab.org">
+    <meta property="og:title" content="Community-toolbox">
```

Comment:
@subhahu123 Did you want to add the dash here to Community toolbox? In search it might make sense to not have that and just have content="Community Toolbox"> 

Referencing from the image sequencer merged PR publiclab/image-sequencer#863

What are your thoughts??